lib/rester/rspec.rb

```diff
@@ -37,67 +37,71 @@
   end
 
   config.before :each, rester: // do |ex|
-    # Gather the request args from the spec descriptions
-    #
-    # For example:
-    #
-    # describe '/v1/tests' do
-    #   context 'GET' do
-    #     context 'With some context' do
-    #     end
-    #   end
-    # end
-    #
-    # would produce:
-    #
-    # context, verb, path = ['With some context', 'GET', '/v1/tests']
-    context, verb, path = ex.example_group.parent_groups.map { |a|
-      a.description unless a.metadata[:description] == a.described_class.to_s
-    }.compact
-
-    begin
-      spec = @rester_stub[path][verb][context]
-    rescue NoMethodError
-      fail Rester::Errors::StubError,
-        "Could not find path: #{path.inspect} verb: #{verb.inspect} context: "\
-          "#{context.inspect} in #{@rester_stub_filepath}"
-    end
-
-    ##
-    # Raw response from the service.
-    # [HTTP CODE, JSON String]
-    ex.example_group.let(:raw_service_response) {
-      @rester_adapter.request(verb.downcase.to_sym, path, spec['request'])
-    }
+    unless ex.pending?
+      # Gather the request args from the spec descriptions
+      #
```

Comment:
how about just `return if ex.pending?` since there's so much else in this block.  No need for it to all be tabbed over.